### PR TITLE
retreive and get enum filter values

### PR DESCRIPTION
**Pull request by ghazwarhili**

### CODE REVIEW

File: src/components/computing-status/computing-type.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@
  */
 
 export enum ComputingType {
-    LOADFLOW = 'LOADFLOW',
+    LOAD_FLOW = 'LOAD_FLOW',
```
Comment (anistouri) [inline src/components/computing-status/computing-type.ts:9]: Why this change?

Comment (ghazwarhili) [reply src/components/computing-status/computing-type.ts:9]: to be coherent with the computing Status enum values

**Review by anistouri — CHANGES_REQUESTED** [review]
In the violation type filter in the voltage violation tab, we should not display the CURRENT option in the list

File: src/services/study/index.js [hunk 2]
```diff
@@ -313,3 +313,20 @@ export function getServersInfos() {
...
     });
 }
+
+export function fetchAvailableFilterEnumValues(
+    studyUuid,
+    nodeUuid,
+    computingType,
+    filterEnum
+) {
+    console.info('fetch available filter values');
```
Comment (Mathieu-Deharbe) [inline src/services/study/index.js:323]: This log could provide some more info (computingType and filterEnum). (Personally I don't find those very useful but I can see they are everywhere so I guess this is a gridsuite convention)

File: src/services/study/index.js [hunk 3]
```diff
@@ -313,3 +313,20 @@ export function getServersInfos() {
...
+    const url = `${getStudyUrlWithNodeUuid(
+        studyUuid,
+        nodeUuid
+    )}/computation/result/enum-values?computingType=${encodeURIComponent(
+        computingType
+    )}&enumName=${encodeURIComponent(filterEnum)}`;
```
Comment (Mathieu-Deharbe) [inline src/services/study/index.js:329]: I think you can renam this `filterEnum` into `enumName` as well.

File: src/components/computing-status/computing-type.ts [hunk 4]
```diff
@@ -6,10 +6,11 @@
...
     SECURITY_ANALYSIS = 'SECURITY_ANALYSIS',
     SENSITIVITY_ANALYSIS = 'SENSITIVITY_ANALYSIS',
     NON_EVACUATED_ENERGY_ANALYSIS = 'NON_EVACUATED_ENERGY_ANALYSIS',
+    SHORT_CIRCUIT = 'SHORT_CIRCUIT',
     ALL_BUSES_SHORTCIRCUIT_ANALYSIS = 'ALL_BUSES_SHORTCIRCUIT_ANALYSIS',
     ONE_BUS_SHORTCIRCUIT_ANALYSIS = 'ONE_BUS_SHORTCIRCUIT_ANALYSIS',
```
[the remark this answers is not in the document]
Comment (ghazwarhili) [reply src/components/computing-status/computing-type.ts:15]: to be coherent with the Enum computing status on the back side

File: src/services/study/index.js (hunk 2 above)
Comment (ghazwarhili) [reply src/services/study/index.js:323]: okey

**Review by anistouri — APPROVED** [review]
